### ISSUE TITLE
No web UI

### ISSUE THREAD

**Issue by remynollet** [conduktor/conduktor-platform#18]
Trying the example-local, there is no UI looking http://localhost:80

How can we access to the console ?

**Comment by remynollet** [comment]
It seems that our zscaler proxy on our laptop is allready use the port 80.
I try to map 
```
  conduktor-platform:
    image: conduktor/conduktor-platform:latest
    #network_mode: "bridge"
    ports:
      - 8000:80
```
the http://localhost:8000 is redirect to http://localhost/home

And 
```
HTTP Error 404. The requested resource is not found.
```

**Comment by remynollet** [comment]
Using http://localhost:8000/home/ works...

**Comment by rokh-conduktor** [comment]
Hello Remy,

Thanks for your feedback and investigation, we will add documentation on port customization.

**Comment by remynollet** [comment]
Thanks

Le ven. 9 sept. 2022 à 14:03, rokh-conduktor ***@***.***> a
écrit :

**Comment by rokh-conduktor** [comment]
Fixed in 1.0
